File: 2137-final-value-of-variable-after-performing-operations/solution.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int i =0;
        for (String j:operations){
            if(j.equals("X++") || j.equals("++X")){
                i+=1;
            }
            else if (j.equals("--X") || j.equals("X--")){
                i-=1;
            }
        }
        return i;
    }
}

File: 2137-final-value-of-variable-after-performing-operations/solution.py
class Solution(object):
    def finalValueAfterOperations(self, operations):
        x=0
        for i in operations :
            if (i== "--X" or i =="X--"):
                x-=1
            elif(i=="X++" or i=="++X"):
                x+=1
        return x
        """
        :type operations: List[str]
        :rtype: int
        """
        
